### DISCUSSION TITLE
https://directus.io/cloud/ redirect here

### DISCUSSION THREAD

**Discussion by gregflln** [directus/directus#14230] [Feature Requests]
https://directus.io/cloud/

Get started button redirect here ??

**Comment by jamescammarano** [comment]
Are you are talking about the `Get Started For Free` button on the homepage that redirects to https://directus.cloud/login

**Reply by rijkvanzanten** [reply]
No, the "Get Started" button on https://directus.io/cloud/ redirects to a new discussion (cc @benhaynes)

**Comment by rijkvanzanten** [comment]
Thanks for the heads up, fix pushed up 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
